### PR TITLE
Add multi-main setup

### PR DESCRIPTION
**Pull request by ivov**
https://linear.app/n8n/issue/PAY-934/documentation-for-multiple-main-instances-mode

**Important:** Please do not merge until announced.

### CODE REVIEW

**Review by StarfallProjects — APPROVED** [review]

File: docs/hosting/scaling/queue-mode.md [hunk 1]
```diff
@@ -171,3 +171,46 @@ You can define the number of jobs a worker can run in parallel by using the `con
...
+
+In queue mode you can run more than one `main` process for high availability.
+
+In a single-mode setup, the `main` process is responsible for:
+
+- Running the API
+- Serving the UI
+- Listening for webhooks
+- Handling manual executions
+- Running triggers and pollers
```
Comment (krynble) [inline docs/hosting/scaling/queue-mode.md:190]:
```suggestion
- Running non-HTTP related triggers, such as timers, persistent connections like RabbitMQ and IMAP
```

File: docs/hosting/scaling/queue-mode.md [hunk 2]
```diff
@@ -171,3 +171,46 @@ You can define the number of jobs a worker can run in parallel by using the `con
...
+- Handling specific licensing tasks
+- Pruning executions and binary data
+
+In a multi-main setup, every `main` process runs the API, serves the UI, listens for webhooks, and handles manual executions. n8n automatically designates a leader `main` process, which is the process in charge of running triggers and pollers, handling specific licensing tasks, pruning executions and binary data. All non-leader `main` processes are known as followers.
```
Comment (krynble) [inline docs/hosting/scaling/queue-mode.md:194]:
```suggestion
In a multi-main setup, every `main` process runs the API, serves the UI, listens for webhooks, and handles manual executions. n8n automatically designates a leader `main` process, which is the process in charge of running any non-HTTP triggers, handling specific licensing tasks, pruning executions and binary data. All non-leader `main` processes are known as followers.
```

File: styles/Vocab/default/accept.txt [hunk 3]
```diff
@@ -44,6 +44,8 @@ onboarding
 Pipedrive
 Pinecone
 Pluralsight
+Pollers
+pollers
```
Comment (krynble) [inline styles/Vocab/default/accept.txt:48]:
```suggestion
```

**Review by krynble — APPROVED** [review]
Just a suggestion, feel free to disregard, but I think people in general won't understand `poller or trigger` so it makes sense to describe them as `non-HTTP triggers`

**Comment by ivov** [comment]
@krynble Sounds good! I'll leave @StarfallProjects to decide.

**Comment by krynble** [comment]
@StarfallProjects this PR should be good to merge, the feature is running stable and we'll be making it available to public now.

**Pull request merged**